index.html

```diff
@@ -0,0 +1,353 @@
+
+<img src="images/horizontal-fold.svg" height="150px" alt="Horizontal fold">
+<img src="images/vertical-fold.svg" height="150px" alt="Vertical fold">
+
+*<br>An illustration of two [=foldable-screen device=] concepts in two different [=postures=]. ([Source](https://w3c.github.io/screen-fold/))*
+
+<aside class="note">
+Traditional single-screen laptop form factor is excluded from the foldable device definition and web specifications that refer to this form factor specifically refer to it as a <dfn>traditional laptop</dfn>. Wikipedia has a more nuanced definition of laptop types, but for the purpose of [related web specifications](#related-web-specifications) [all laptop types](https://en.wikipedia.org/wiki/Laptop#Types) are considered traditional laptops.
+</aside>
+
+### Dual-screen device
+
+A <dfn data-export="">dual-screen device</dfn> has two or more physical internal screens made of solid materials placed together to form a larger foldable virtual screen. A [=dual-screen device=] is an instance of a <dfn data-export="">multi-screen device</dfn> with two screens. A defining characteristic of the [=dual-screen device=] form factor is a physical hinge that can rotate from 0 degree to at most 360 degrees that forms a physical gap in between the screens. The range of the hinge movement as well as the width of the gap varies depending on the hardware design. The screens separated by the physical hinge are made of solid materials, typically glass, and as such are not flexible. This design requires web developers to work around the fact there is a physical gap between the screens on both sides of the fold.
+
+### Foldable-screen device
+
+A <dfn data-export="">foldable-screen device</dfn> has one or multiple screens made of foldable material that can bend so that the each screen can fold onto itself. The hinge can typically rotate from 0 degree to 180 degrees depending on the flexible screen materials’ capability of sustaining the bend radius. The material around the area of the screen that can bend is often softer than on the other parts of the screen. To differentiate from a dual-screen device, a foldable-screen device does not have a physical gap between the screens that form the logical regions. This makes it easier for existing web content to be presented without visual distractions when the foldable-screen device is folded open into so-called flat posture.
+
+### Multi-monitor setups
```

Comment:
I wonder if we should use the term "multi-screen device", "extended-screen device", or similar instead. Fine to leave it as-is for now...

Reply:
Not sure if multi-screen works here since this category is specifically for devices that fold, and while some foldables can be multiscreen not all multiscreen are foldables.

Reply:
"Multi-monitor setup" was derived from https://en.wikipedia.org/wiki/Multi-monitor If there is a better established term we should switch to that. It sounds like we should keep the "multi-monitor setup" for now.

I noted "extended-screen device" as a synonym, but should settle on a canonical term for the definition.

Reply:
Sounds good, thanks!